docs/getting-started/coverage.md

```diff
@@ -0,0 +1,53 @@
+# Scanning Coverage
+
+## Programming languages
+Trivy detects code dependencies for the purpose of SBOM generation and vulnerability detection. The following programming languages have some level of support:
+
+- Ruby
+- Python
+- PHP
+- Node.js
+- .NET
+- Java
+- Go
+- Rust
+- C/C++
+- Elixir
+- Dart
+- Swift
+
+Some features might be not be universally supported. For a full supportability matrix, [see here](../docs/vulnerability/detection/language.md).
+
+## OS
+Trivy detects installed packages for the purpose of vulnerability detection. The following package managers are supported:
+
+- Alpine Linux
+- Wolfi Linux
+- Red Hat Universal Base Image
+- Red Hat Enterprise Linux
+- CentOS
+- AlmaLinux
+- Rocky Linux
+- Oracle Linux
+- CBL-Mariner
+- Amazon Linux
+- openSUSE Leap
+- SUSE Enterprise Linux
+- Photon OS
+- Debian GNU/Linux
+- Ubuntu
+- Distroless
+
+Some features might be not be universally supported. For a full supportability matrix, [see here](../docs/vulnerability/detection/os.md).
+
+## IaC & configuration
+Trivy reads IaC & configuration languages for the purpose of misconfiguration detection and custom checks.
+
+- Kubernetes
+- Dockerfile, Containerfile
+- Terraform 
+- CloudFormation
+- Azure ARM Template
+- Helm Chart
```

Comment:
I was going to write a similar coverage page but then saw your PR :) 

My observation was that should we also mention which providers we support? For instance currently we support, AWS, GCP and Azure IaC scanning but if tomorrow (there's a [WIP](https://github.com/aquasecurity/defsec/pull/1242) here) we add support for another provider (another Cloud) there doesn't seem to be a way to be able to mention that.

Reply:
I just collected data from https://aquasecurity.github.io/trivy/v0.38/docs/misconfiguration/policy/builtin/. is there anything missing there?
also, not sure what you meant that there doesn't seem to be a away to add another provider, WDYM?

Reply:
@simar7 Can you take another look at Itay's answer?

Reply:
actually lgtm now, I forgot what I had in mind back then so will save it for later :)